### ISSUE TITLE
beautifultable is not beautiful for multibyte string

### ISSUE THREAD

**Issue by showwin** [pri22296/beautifultable#39]
Hi, 
I'm trying to print multibyte string in beautifultable. But the result is not beautiful, unsuitable number of space.

like this:
```python
from beautifultable import BeautifulTable
table = BeautifulTable()
table.column_headers = ['Language', 'Hello']
table.append_row(['Japanese', 'こんにちは'])
table.append_row(['English', 'Hello'])
table.append_row(['Chinese', '你好'])
table.append_row(['Emoji', '😉'])
print(table)
```
```
+----------+-------+
| Language | Hello |
+----------+-------+
| Japanese | こんにちは |
+----------+-------+
| English  | Hello |
+----------+-------+
| Chinese  |  你好   |
+----------+-------+
|  Emoji   |   😉   |
+----------+-------+
```

Is there any way to solve this problem?
Thanks.

```
$ pip freeze | grep beautifultable
beautifultable==0.5.3
```

**Comment by pri22296** [comment]
Yeah, beautifultable currently doesn't support multibyte strings. I could take a look at if this can be implemented however can't give you any ETA on this.

**Comment by pri22296** [comment]
Implemented in 5de5af4
